src/components/Contentful/JourneyCardCarousel.vue

```diff
@@ -6,23 +6,12 @@
 		>
 			Take the <u>next step</u> on your impact journey
 		</h2>
-		<KvCarousel
-			:key="orderedSlides.length"
-			:embla-options="{
-				loop: false,
-				align: 'start',
-			}"
-			:slide-max-width="singleSlideWidth"
-			:multiple-slides-visible="true"
-			class="journey-card-carousel tw-w-full tw-overflow-visible"
-			@change="handleChange"
-		>
-			<template
-				v-for="(slide, index) in orderedSlides"
-				#[`slide${index}`]
-				:key="index"
-			>
+		<template v-if="inLendingStats">
+			<div class="tw-w-full tw-h-full tw-flex tw-flex-col md:tw-flex-row lg:tw-flex-nowrap tw-gap-4">
 				<MyKivaCard
+					v-for="(slide, index) in orderedSlides"
```

Comment:
I think we might want to keep this in the carousel regardless, since we're now getting wrap in some screen sizes. The carousel should be able to handle all screen sizes:
<img width="1068" height="848" alt="image" src="https://github.com/user-attachments/assets/9fbd9a7c-3609-48a9-85f7-8f138328aaf4" />

Reply:
Isn't the goal to have the 3 cards in a row? Using the carousel for all 3 might be a simple solution to ensure there's no wrap for different screen sizes.

Reply:
For Shua's account, I'm no longer seeing the next step cards (shown in previous screenshot). Is that expected? I assumed I'd still be seeing the same 3 cards.
<img width="1068" height="556" alt="image" src="https://github.com/user-attachments/assets/e04d0fe3-328c-486d-b7d4-4032055ff8f6" />

Reply:
Sent in Slack